featurebyte/service/online_serving.py

```diff
@@ -426,15 +423,19 @@ async def get_request_code_template(
             raise UnsupportedRequestCodeTemplateLanguage("Supported languages: ['python', 'sh']")
 
         # construct entity serving names
-        entity_serving_names = await self.feature_list_service.get_sample_entity_serving_names(
-            feature_list_id=feature_list.id,
-            count=1,
+        assert deployment.serving_entity_ids is not None
```

Comment:
Should we either fallback to the old logic or raise a 422 error when serving_entity_ids is not set? I think it the field should be available for all new deployments.

Reply:
Indeed. I will add fallback handling